spec/models/task_spec.rb

```diff
@@ -27,6 +27,58 @@
 
   end
 
+  describe "" do
```

Comment:
maybe, just name it `.scopes`, or something like that :question: 
but happy if you merge it as it is now